src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/TraceLogging/TraceLoggingEventSource.cs

```diff
@@ -440,9 +440,6 @@ public unsafe void Write(string? eventName, EventSourceOptions options)
                 descriptors[1].SetMetadata(pMetadata1, nameInfo.nameMetadata.Length, 1);
                 descriptors[2].SetMetadata(pMetadata2, eventTypes.typeMetadata.Length, 1);
 
-#if (!ES_BUILD_PCL)
```

Comment:
This file is built as OOB package for .NET Framework. (`src\libraries\Microsoft.Diagnostics.Tracing.EventSource.Redist\src\Microsoft.Diagnostics.Tracing.EventSource.Redist.csproj`)

I think this should stay, perhaps under a different ifdef (`#if ES_BUILD_STANDALONE` ?). Make sure to fix all places in this file.

Reply:
I connected with @noahfalk on this. We can't find the definition of `ES_BUILD_PCL` but we see it's referenced in some other places too.  We agreed that for this PR, I should:

1. Change this to `#if (!ES_BUILD_PCL && !NETCOREAPP)`
1. Submit an issue to the `area-System.Diagnostics.Tracing` area to investigate the potentially stale `ES_BUILD_PCL` references

Reply:
I filed https://github.com/dotnet/runtime/issues/39412 to investigate the potentially unused constant.